paasta_tools/cli/cmds/spark_run.py

```diff
@@ -532,6 +534,21 @@ def get_spark_config(
     # Limit a container's cpu usage
     non_user_args['spark.mesos.executor.docker.parameters'] += ',cpus={}'.format(user_args['spark.executor.cores'])
 
+    if 'spark.eventLog.enabled' not in user_args:
```

Comment:
I would leave spark.eventLog.enabled in user_args with default true so that users can disable event logging if they want to.